Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1558,6 +1559,19 @@ int CLuaElementDefs::IsElementFrozen(lua_State* luaVM)
     return 1;
 }
 
+bool CLuaElementDefs::IsElementOnGround(CClientEntity* entity) {
```

Comment:
Although an exception can be thrown, all code paths must return a boolean.

Reply:
The exception is probably the reason why its not compiling